src/test/java/org/wiremock/integrations/testcontainers/WireMockContainerBannerTest.java

```diff
@@ -21,7 +21,7 @@
 
 class WireMockContainerBannerTest {
 
-    WireMockContainer wireMockContainer = new WireMockContainer("2.35.0");
+    WireMockContainer wireMockContainer = new WireMockContainer(WireMockContainer.DEFAULT);
```

Comment:
IDK how this will be handled but just sharing a little bit of Testcontainers experience.  Testcontainers recommend to let the users declare the version and update on their on peace because the DEFAULT value can change at any time and can cause errors or even breaking changes which will be unexpected by the user.

Reply:
> Testcontainers recommend to let the users declare the version and update on their on peace because the DEFAULT value can change at any time and can cause errors or even breaking changes which will be unexpected by the user.

It is fair assumption. We expect the breaking changes to happen only between major releases of WireMock, and of course they will be annotated as such. I can rename `DEFAULT_VERSION` to something more explicit so that users understand the concept. Javadoc otherwise?

Reply:
Done